coordinator/src/main/java/com/tencent/rss/coordinator/CoordinatorServer.java

```diff
@@ -100,15 +99,20 @@ public void stopServer() throws Exception {
     if (accessManager != null) {
       accessManager.close();
     }
+    if (clientConfManager != null) {
+      clientConfManager.close();
+    }
     server.stop();
   }
 
-  private void initialization() throws FileNotFoundException {
+  private void initialization() throws Exception {
     this.applicationManager = new ApplicationManager(coordinatorConf);
 
     ClusterManagerFactory clusterManagerFactory = new ClusterManagerFactory(coordinatorConf);
     this.clusterManager = clusterManagerFactory.getClusterManager();
-
+    if (coordinatorConf.getBoolean(CoordinatorConf.COORDINATOR_DYNAMIC_CLIENT_CONF_ENABLED, false)) {
```

Comment:
the manager should always be initialized, and it returns empty if COORDINATOR_DYNAMIC_CLIENT_CONF_ENABLED = false

Reply:
ok